scripts/RuntimeArguments+Generated.erb

```diff
@@ -0,0 +1,110 @@
+
```

Comment:
Fix whitespace

Reply:
This is consistent with the code style it was copied from.